ara/clients/utils.py

```diff
@@ -20,14 +22,18 @@ def get_client(
     if username is not None and password is not None:
         auth = HTTPBasicAuth(username, password)
 
+    cert = None
```

Comment:
Is this right ? get_client would default ``cert`` to ``None`` and this line would set it to ``None`` even if there is another value passed.

Reply:
Nice catch! My intention was just to set a default value to be passed to `AraHttpClient`.